server/graphql/invoice_line/src/mutations/outbound_shipment_line/unallocated_line/update.rs

```diff
@@ -26,6 +26,7 @@ pub struct UpdateInput {
 #[graphql(field(name = "description", type = "String"))]
 pub enum UpdateErrorInterface {
     RecordNotFound(RecordNotFound),
+    ForeignKeyError(ForeignKeyError),
```

Comment:
Is this an internal error? or bad user input? if it should be a structured error, maybe add a docs comment about what it means.